lib/providers/events.dart

```diff
@@ -8,6 +9,10 @@ class Events extends ChangeNotifier {
   List<Event> _allEvents = [];
   List<Event> _events = [];
   List<Event> get events => [..._events];
+  List<Event> get todayEvents => [
+        ..._events.where((event) => event.formattedLocalDateTime
+            .contains(DateFormat('EEE, MMM d').format(DateTime.now())))
```

Comment:
COOOL!

didn't know `.contains` stuff at all
`DateFormat('EEE, MMM d')` part could be a constant above the class tho since it never changes

Reply:
Agree